config_test.go

```diff
@@ -32,7 +32,8 @@ var testConfig = `{
   "PollIntervalSeconds": 60,
   "ServerURL": "mender.io",
   "DeviceID": "1234-ABCD",
-  "ServerCertificate": "/data/server.crt"
+  "ServerCertificate": "/var/lib/mender/server.crt",
```

Comment:
Actually the certificate is somewhere in ${sysconfdir}/mender in the image. Though I guess it does not really matter which path we end up using in tests.